packages/block-library/src/term-description/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { termDescription as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+
+const { name } = metadata;
+export { metadata, name };
+
+export const settings = {
+	title: _x( 'Term Description', 'block title' ),
```

Comment:
We should add a small description here probably.

Reply:
I don't think it matters, because as soon as we move all translations to `block.json` the context is going to be generalized.

Reply:
Sorry - I made this confusing :) 

I meant adding a `description` in the block to show in Block Card :)

Reply:
Added 👍